File: Data Structures & Algorithms/duplicate-integer/submission-2.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int n = nums.size();
        set<int> s;
        for(int i = 0; i < n; i++){
            s.insert(nums[i]);
        }

        if(s.size() == nums.size()){
            return false;
        }
        return true;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        const n = nums.length;
        const result = new Array();
        for(let i = 0; i < n - 1; i++) {
            for(let j = i+1; j < n; j++){
                if(nums[i] + nums[j] === target) {
                    result.push(i, j);
                }
            }
        }
        return result;
    }
}
